### DISCUSSION TITLE
No longer detecting fans

### DISCUSSION THREAD

**Discussion by Frem22** [Rem0o/FanControl.Releases#1595] [Q&A]
Since the last update V148 (or possibly the one prior) FanControl has been failing to find any fans on my system. (I was getting a popup advising that it failed to <cant remember sry>? and at the bottom it informed me that if I had auto load then try increasing the delayed startup)

After fiddling with some settings and saving I no longer get the failure popup I described above but.. Nothing on the screen is populated anymore except the curves section.

When I click on the 'Controls' setup and fire any of the pairing routines I get no errors but also get no Controls showing up afterward either.

**Comment by Rem0o** [comment]
Try a clean install in a new folder on your computer, and report what you get.

**Comment by Frem22** [comment]
The wizard started off normally testing fans (there was no changes to any fans while it was testing BTW) and after it completed there was nothing populated except 'Flat curve example'.

**Reply by Rem0o** [reply]
Motherboard? There is an issue with Asrock boards right now, should be fixed soon.

**Reply by tangknee** [reply]
> Motherboard? There is an issue with Asrock boards right now, should be fixed soon.

I'm having a similar issue, seems to have started with v148. I also have an Asrock board.

**Reply by Frem22** [reply]
> Motherboard? There is an issue with Asrock boards right now, should be fixed soon.

Asrock Steel Legend x670e

**Comment by jamesnmandy** [comment]
I am suddenly also having a problem with 1.49 on a Gigabyte Aorus M B450 motherboard.  The other day it was detecting the sys fan headers that my fan controller is connected to.  Today it doesn't even recognize that header.  Getting the fans with a ? saying it's being overridden by an external source.  I didn't touch a thing.  Those fans will not respond to a "Mix" fan curve setup anymore either.
